### ISSUE TITLE
java.lang.RuntimeException: Unable to instantiate receiver

### ISSUE THREAD

**Issue by jajangjayus** [sikandernoori/time_change_detector#2]
Hello, I faced error java.lang.RuntimeException: Unable to instantiate receiver, when i changed device time zone

I assumed the problem in file manifest

```
<receiver
      android:name=".TimeChangeDetectorPlugin"
      android:exported="true">
      <intent-filter>
          <action android:name="android.intent.action.TIME_SET" />
          <action android:name="android.intent.action.TIMEZONE_CHANGED" />
          <action android:name="android.intent.action.DATE_CHANGED"/>
      </intent-filter>
  </receiver>
```

please tell me what should i do, thanks

**Comment by Hideart** [comment]
@jajangjayus 
In my case, the solution was to replace
```
<receiver
      android:name=".TimeChangeDetectorPlugin"
      ...
```
with
```
<receiver
      android:name="com.randomforest.time_change_detector.TimeChangeDetectorPlugin"
```

**Comment by Mamasodikov** [comment]
> @jajangjayus In my case, the solution was to replace
> 
> ```
> <receiver
>       android:name=".TimeChangeDetectorPlugin"
>       ...
> ```
> 
> with
> 
> ```
> <receiver
>       android:name="com.randomforest.time_change_detector.TimeChangeDetectorPlugin"
>       ...
> ```

Based on this https://github.com/sikandernoori/time_change_detector/pull/6#issue-1925978214

**Comment by sikandernoori** [comment]
Fix in version 0.0.3
